assets/javascript/game.js

```diff
@@ -0,0 +1,195 @@
+    },
+    correctAnswer: "d",
+    explanation:"A Deeper Understanding is the fourth studio album by American indie rock band The War on Drugs. It was released on August 25, 2017, through Atlantic Records. ",
+    
+  },
+  {
+    question: "Who won best alternative music album in 2018 Grammys?",
+    answers: {a: "Everything Now - Arcade Fire",b: "Humanz - Gorillaz",c: "American Dream - LCD Soundsystem",d: "Sleep Well Beast - The National"
+    },
+    correctAnswer: "d",
+    explanation:"Sleep Well Beast is the seventh studio album by American indie rock band The National, released on September 8, 2017 through 4AD.",
+    
+  },
+ 
+];
+
+
+var answer = "";
+var triviaQuestion = "";
+
+
+var unanswered = 0;
+var timer;
+var index = 0;
+var timeKeeper;
+
+var button;
+var secs = 30;
+var countdownBoolean = false;
+
+var correctAnswers = 0;
+var wrongAnswers = 0;
+
+
+
+function answerSentence() {
+    $("#triviaQuestion").empty();
+  
+    if (answer === "correct") {
+      triviaQuestion = "Correct!";
+    } else if (answer === "zero") {
+      triviaQuestion = "Sorry, you ran out of time!";
+    } else if (answer === "wrong") {
+      triviaQuestion = "Sorry, thats not correct!";
+    }
+  
+    $("#triviaQuestion").text(triviaQuestion);
+  }
+
+function displayQuestion() {
+  $(".answerBox").hide();
+  $(".scoreBoardBox").hide();
+  $(".triviaBox").show();
+
+  $("#triviaQstn").text(triviaQuestions[index]["question"]);
+
+  for (var key in triviaQuestions[index]["answers"]) {
+    button = $("<button>");
+    var buttonText = triviaQuestions[index]["answers"][key];
+    var buttonClass = "btn btn-secondary btn-lg btn-block answerButtons";
+
+    button.text(buttonText).addClass(buttonClass).attr("data-val", key);
+
+    $(".answersButtons").append(button);
+  }
+
+  startCountdown();
+}
+
+$("#timeLeft").text(secs);
+
+function countdown() {
+    secs--;
+    $("#timeLeft").text(secs);
+  
+    if (secs == 0) {
+      answer = "zero";
+      unanswered++;
+      
+      answerSentence();
+      showAnswer();
+      countdownStop();
+    }
+  }
+  
+  function startCountdown() {
+    if (!countdownBoolean) {
+      timeKeeper = setInterval(countdown, 1000);
+      countdownBoolean = true;
+    }
+  }
+  
+  function countdownStop() {
+    countdownBoolean = false;
+    clearInterval(timeKeeper);
+    secs = 25;
+    $("#timeLeft").text(secs);
+  }
+
+function showAnswer() {
+  $(".answerBox").show();
+  $(".triviaBox").hide();
+
+  var answerLetter = triviaQuestions[index]["correctAnswer"];
+
+  $("#answer").text(
+    "The answer is: " + triviaQuestions[index]["answers"][answerLetter]
+  );
+  $("#answerExplanation").text(triviaQuestions[index]["explanation"]);
+  
+
+  index++;
+  $(".answersButtons").empty();
+
+  clearInterval(timer);
+  timer = setTimeout(isFinalResult, 4000);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['isFinalResult' was used before it was defined. (no-use-before-define)](https://app.codacy.com/app/columbia-fsf-f2019/trivia-game/pullRequest?prid=4555491)